addons/minio/template/base/install.sh

```diff
@@ -8,12 +8,35 @@ function minio_pre_init() {
     if [ -z "$MINIO_CLAIM_SIZE" ]; then
         MINIO_CLAIM_SIZE="10Gi"
     fi
+
+    # verify if we need to migrate away from the deprecated 'fs' format.
+    local minio_replicas
+    minio_replicas=$(kubectl get --no-headers deploy minio -n "$MINIO_NAMESPACE" -o custom-columns=:.spec.replicas 2>/dev/null)
```

Comment:
we have a pattern of using -o jsonpath or --template. can you use that instead?

Reply:
fixed.